NewForce-Cohort-9/newforce-c9-tabloid-full-stack-TabloidFullStack
# Post List - ticket 3

### Description

Created Posts page to view all posts in a list displaying the title, author, and category.
See [Trello Ticket #3](https://trello.com/c/VX7oPWJz)

## Type of change


### Testing Instructions

1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout `ml-PostList-ticket3`
5. If you don't see the most recent version of my files, git pull origin `ml-PostList-ticket3`
6. If you haven't created the Tabloid MVC database, please do so:

- Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/newforce-c9-tabloid-full-stack-TabloidFullStack/blob/main/SQL/01_Tabloid_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/newforce-c9-tabloid-full-stack-TabloidFullStack/blob/main/SQL/02_Tabloid_Seed_Data.sql)

8. Click the execute with debugger solid green button to launch the app
9. Run `npm start` from the client/tabloid directory in the terminal
10. In browser, please go to Login on the nav bar and log in with this email: [foo@bar.com] and random password (doesn't matter)
11. On the home page - Click the Posts menu option
13. When they select the Posts link in the menu, they should be directed to a page that lists all the Posts in order of publication date with the most recent on top

# Checklist:
